packages/azure-themes/src/azure/Constants.ts

```diff
@@ -5,6 +5,7 @@ import {
   getScreenSelector,
 } from '@fluentui/react/lib/Styling';
 
+export const inputHeight = '18px';
```

Comment:
Thank you for adding this, as we will be 18px width / height for multiple controls (checkbox, radio buttons)